object_manager.go

```diff
@@ -317,20 +336,35 @@ func (objMgr *ObjectManager) GetFixedAddress(netview string, cidr string, ipAddr
 	return &res[0], nil
 }
 
-func (objMgr *ObjectManager) UpdateFixedAddress(fixedAddrRef string, macAddress string, name string, vmID string) (*FixedAddress, error) {
+func (objMgr *ObjectManager) GetFixedAddressFromref(ref string) (*FixedAddress, error) {
+        fixedAddr := NewFixedAddress(FixedAddress{})
+        err := objMgr.connector.GetObject(fixedAddr, ref, &fixedAddr)
+        return fixedAddr, err
+}
+
+func (objMgr *ObjectManager) DeleteFixedAddress(ref string) (string, error) {
+        return objMgr.connector.DeleteObject(ref)
+}
+
+func (objMgr *ObjectManager) UpdateFixedAddress(fixedAddrRef string, matchclient string, macAddress string, vmID string, vmName string) (*FixedAddress, error) {
```

Comment:
what does `matchclient` do ?

Reply:
so , when we create a Reservation by giving 00:00:00:00:00:00 using create fixed address call and update this reserved address to fixed address then the match client flag should be enabled.